printer_mods/README.md

```diff
@@ -25,6 +25,7 @@ like so:
 
 | Creator | Mod title | Description | V0 | V1.8 | V2.4 | VSW | Trident |
 | --- | --- | --- | --- | --- | --- | --- | --- |
+|Simone67|[Tornado Air Flow](./Simone67/TornadoAirFlow)| Use two 5015 fan to get a super airflow|:x:|:x:|:heavy_check_mark:|:x:|:x:|
```

Comment:
This should be moved down to it's position in Alphabetical ordering, not at the top.

Reply:
You indicated this is only for V2.4, and not compatible with any other printer (with X).

Why wouldn't this work on any other printer that has an Afterburner? Like the V1.8, Trident, and Switchwire?

Reply:
Thanks for your comments, I will fix anyone..

Reply:
I did it..